src/hooks/useHandleAppTileEvents.ts

```diff
@@ -59,12 +59,32 @@ export const useHandleAppTileEvents = () => {
     });
   };
 
+  const openCircleDiscussionScreen = (circleId: string, circleName: string) => {
+    const circleTile: CircleTile = {
+      circleId,
+      circleName,
+      buttonText: circleName,
+      isMember: true,
```

Comment:
I don't believe the `isMember` field is used anywhere. Using `true` for now as the program users should be enrolled in the circle linked to the program activity.

Reply:
Happy to delete this field altogether if I can get confirmation that we don't need it anymore.

Reply:
Yeah I don't think we need this